cmd/delete/domain/domain.go

```diff
@@ -40,6 +39,10 @@ func NewdomainCmd() *cobra.Command {
 				return err
 			}
 			urls := strings.Split(p.Callbackurl, ";")
+			if len(urls) == 1 {
+				fmt.Println("Cannot delete the last domain")
+
+			}
```

Comment:
We should return from here if there is only one domain right, ideal way is to return the error
```
return &cmdutil.FlagError{Err: errors.New("Cannot delete the last domain")}
```